### PR TITLE
Modularized project

### PR DESCRIPTION
**Pull request by ArnyminerZ**
Done:
- Tests are passing
- Dokka is working

To do:
- Develop sample app (maybe in different PR?)

### CODE REVIEW

**Comment by ArnyminerZ** [comment]
Well, there are a lot of changes, but because everything had to be moved to a `lib` module. Basically the only things I've changed are:
- Created two modules:
  - `lib`: exactly the same as we had before
  - `sample-app`: a sample app to showcase how to use the library
- [`README.md`](https://github.com/bitfireAT/cert4android/blob/master-new/README.md): Changed "How to use" to use the Jitpack library.
- [`lib/build.gradle`](https://github.com/bitfireAT/cert4android/blob/master-new/lib/build.gradle): Added `android.defaultConfig.aarMetadata`, enabled `android.testFixtures`, configured `android.publishing` and `publishing`.
  Technically, with the current configuration, we could publish to Maven Central if we ever want, however, Jitpack only uses the local build, so no need to configure anything else.
- [`settings.gradle`](https://github.com/bitfireAT/cert4android/blob/master-new/settings.gradle): Take into account that the module name for the library has been renamed to `cert4android` here. It's recommended by Google to name the module like the library, and this is their recommended approach.

File: build.gradle [hunk 1]
```diff
@@ -1,76 +1,9 @@
...
-
-    androidTestImplementation 'androidx.test:runner:1.5.2'
-    androidTestImplementation 'androidx.test:rules:1.5.0'
-    androidTestImplementation 'com.squareup.okhttp3:mockwebserver:4.10.0'
-
-    testImplementation 'junit:junit:4.13.2'
-}
\ No newline at end of file
+group = 'at.bitfire'
+version = '1.0.0'
```
Comment (rfc2822) [inline build.gradle:9]: Is this the same version as in lib/build.gradle publishing? Do we need both?

**Review by rfc2822 — CHANGES_REQUESTED** [review]
Thanks, that looks very useful already! I have a few (very small) questions but first the important thing:

I have created a button to the sample app that uses the `CustomCertManager`, which seems to work, except that it can't render the Activity when you click the notification (seems that the XML is not included). Can you please have a look and then explicitly re-request review (so that I know that you're finished)?

File: build.gradle (hunk 1 above)
Comment (ArnyminerZ) [reply build.gradle:9]: Yes, both are necessary. There should be a way to put them in the same variable or something, but I can't manage to get it working

**Pull request merged**